esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/index.rst

```diff
@@ -7,3 +7,4 @@ User Guide
 
    quick_start
    workflow
+   adding_a_recipe
```

Comment:
(We could consider creating a "Developer Guide" and add this documentation to the "Developer Guide"; I'd argue this is more for a developer of the RTW, rather than a user.)

Reply:
I've moved this to the front page in commit 615b4a277. Perhaps i should make a discussion on the repo about adding a developer guide?

Reply:
@mo-gill and I discussed this at the stand-up this morning and opened #3658, which includes consideration of whether to create a "Developer Guide" section within the RTW documentation 👍